examples/simple/api/complex.rs

```diff
@@ -19,10 +19,10 @@ async fn main() -> Result<(), Error> {
         .map_response(process_response)
         .service(service_fn(handler));
 
-    lambda_runtime::run(handler).await
+    run_service(handler).await
 }
 
-pub async fn handler(_req: Request) -> Result<impl IntoResponse, Error> {
```

Comment:
We should now specify this in the response builder as this was previously handled by `into_response`.

```
header(http::header::CONTENT_TYPE, "application/json")
```